File: ass1/demo01.pl
#!/usr/bin/perl -w
# put your demo script here
print "Enter number x: ";
$x = <STDIN>;
chomp $x;
print "Enter number y: ";
$y = <STDIN>;
chomp $y;
if($x == $y){
	print "by golly you've done it!\n";
} else {
	print "hint: what if they were the same?\n";
	exit;
}
my $num_lines = 1;
print "Enter a few lines now for your chance to win! Win what? I don't know!\n";
print "You say:\n";
while ($line = <STDIN>){
	if ($line =~ /Cat|Dog|dog|cat/) {
		print "I say: I love animals!!\n";
		last;
	} else {
		print "I say: I guess that's an OKAY choice of input I really like fluffy things...\n";
	}
	$num_lines++;
}

print "that only took you $num_lines tries to get\n";

File: ass1/demo01.py
#!/usr/local/bin/python3.5 -u
import sys, re
# put your demo script here
print ("Enter number x: ")

x = sys.stdin.readline()
x = x.rstrip()
print ("Enter number y: ")

y = sys.stdin.readline()
y = y.rstrip()
if x == y:
    print ("by golly you've done it!")

else:
    print ("hint: what if they were the same?")

    sys.exit
num_lines = 0
print ("Enter a few lines now for your chance to win! Win what? I don't know!")

print ("You say:")

for line in sys.stdin:
    if re.match(r'Cat|Dog|dog|cat', line):
        print ("I say: I love animals!!")

        break
    else:
        print ("I say: I guess that's an OKAY choice of input I really like fluffy things...")

    num_lines += 1

print("that only took you %d tries to get" % num_lines)
